app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -270,9 +272,11 @@ public void onConnected(Bundle bundle) {
     }
 
     private void initSummitNotificationHandler() {
-        NotificationHandler notificationHandler = new NotificationHandler(this);
-        if (notificationHandler.shouldSetAlarm()) {
-            notificationHandler.setAlarmForNotification();
+        for (TaggedEventSeries eventSeries : App.getInstance().currentTaggedEventSeries()) {
```

Comment:
Yeah but creation of the alarm with intent and setting it can be costly, I think.